source/overview/faq.rst

```diff
@@ -187,6 +187,31 @@ Does Mattermost have an official website-based plug-in to offer anonymous chat t
 
 Not yet. You can `upvote the feature proposal online <https://mattermost.uservoice.com/forums/306457-general/suggestions/8810731-implement-a-site-chat-feature>`_ to add your support. If you create such a plug-in, we would love to see it open sourced and made available to the community. 
 
+
+Design Decisions
+----------------
+
+Why does Mattermost give an error message when they incorrectly enter a password? 
```

Comment:
You need an antecedent for "they". Also, this isn't really the right question. All sites give error messages when the user enters an incorrect password. Something like "Why does Mattermost disclose whether or not an account exists when a user enters an incorrect password?" would be more accurate.